CHANGES.md

```diff
@@ -177,6 +177,11 @@ Unreleased
   new `(subst <disable|enable>)` stanza to the `dune-project` file.
   (#4864, @kit-ty-kate)
 
+- Simplify the way `dune` discovers the root of the workspace. It now
```

Comment:
Is this `dune` the command or Dune the tool? If it’s the latter, consider capitalizing it for consistency.